ios/RNGestureHandlerManager.m

```diff
@@ -132,25 +128,35 @@ - (void)handleClearJSResponder
 
 #pragma mark Root Views Management
 
-- (void)registerRootViewIfNeeded:(UIView*)childView
+- (void)registerRootViewGestureRecognizerIfNeeded:(UIView*)childView
```

Comment:
we are no longer interested in root views here but rather in views that have touchHandler attached. This naming is now misleading.